static/js/all-alerts.js

```diff
@@ -365,10 +450,26 @@ let alertColumnDefs = [
         width: 100,
         cellRenderer: stateCellRenderer,
     },
+    {
+        headerName: 'Muted For',
```

Comment:
should this column name be changed to `Silenced for` since everywhere else in the UI we are using the word  silence instead of mute